server/core/src/https/views/mod.rs

```diff
@@ -36,7 +37,7 @@ pub fn view_router() -> Router<ServerState> {
         .route("/", get(|| async { Redirect::permanent("/ui/login") }))
         .route("/apps", get(apps::view_apps_get))
         .route("/reset", get(reset::view_reset_get))
-        .route("/update_credentials", get(reset::view_self_reset_get))
+        .route("/update-credentials", get(reset::view_self_reset_get))
```

Comment:
Why was this changed? We use underscores in URLs.

```suggestion
        .route("/update_credentials", get(reset::view_self_reset_get))
```